src/installation/guides/chroot.md

```diff
@@ -0,0 +1,252 @@
+then mount any other filesystems on top of this new root. `/mnt` may be used
+instead of creating a new directory.
+
+This directory will be called `$VDROOT` for the remainder of this guide.
+
+```
+# VDROOT=/mnt
+```
+
+If using UEFI, mount the EFI System Partition as `$VDROOT/boot/efi`.
+
+For example, if `/dev/sda2` is to be mounted as `/` and `/dev/sda3` is to be
+booted as `/home` and `dev/sda1` is the EFI System Partition
+
+```
+# mkdir -p "$VDROOT"
+# mount /dev/sda2 "$VDROOT"/
+# mkdir "$VDROOT"/boot/
+# mount /dev/sda3 "$VDROOT"/home/
+# mkdir -p "$VDROOT"/boot/efi
+# mount /dev/sda1 "$VDROOT"/boot/efi
+```
+
+Initiliazile swap space, if desired using
+[mkswap(8)](https://man.voidlinux.org/mkswap.8).
+
+## Base Installation
+
+[Select a mirror to download packages
+from.](../../xbps/repositories/mirror/index.md) For simplicity, save this URL to
+a shell variable for later use. Append `/current` to this repository url to
+install a glibc distribution, append `/current/musl` for a musl distribution (NB
+musl binaries are not available for i686).
+
+For example,
+
+```
+# REPO=https://alpha.de.repo.voidlinux.org/current
+```
+
+xbps also needs to know what architecture is being installed, available options
+are `x86_64`, `x86_64-musl` and `i686` for PC architecture computers.
+
+For example,
+
+```
+# ARCH=x86_64
+```
+
+This architecture must be compatible with your current operating system, but
+does not need to be the same. If your host is running an x86_64 operating
+system, any of the three architectures can be installed (whether the host is
+musl or glibc), but an i686 host can only install i686 distributions.
+
+Three metapackages are available to boot strap Void: `base-system`,
+`base-voidstrap` and `base-minimal`. `base-minimal` is a strict subset of
+`base-voidstrap` and `base-voidstrap` is a strict subset of `base-system`. You
+may wish to compare these packages for yourself. `xbps-query -x [package]` will
+list dependencies pulled in for the given package. Even `base-system` is a very
+minimal installation.
+
+`base-system` is the package that is installed when using the interactive
+installer. `base-voidstrap` is the package that is installed in rootfs tarballs.
+
+`base-minimal` and `base-voidstrap` do not come with a linux kernel. The `linux`
+package always has the newest available kernel. Alternatively, a long term
+support kernel can be used by installing `linux-lts`.
+
+NB `base-minimal` does not include a
+[terminfo(5)](https://man.voidlinux.org/terminfo.5) database, which will cause
+your terminal to behave in unexpected ways after entering the chroot. You may
+wish to install the package `ncurses-base` to get these files.
+
+If unsure, install `base-system`.
+
+```
+# PKGS="base-system"
+```
+
+Use xbps to boot strap the distribution
+
+```
+# XBPS_ARCH=$ARCH xbps-install -S -r $VDROOT -R "$REPO" $PKGS
+```
+
+## Configuration
+
+### Edit Files
+
+Write the hostname to `$VDROOT/etc/hostname`. Go through options in
```

Comment:
This can all be done inside the chroot and would probably be clearer there.

Reply:
Okay. My intention in doing that outside the chroot was that once inside the chroot the only text editor available is an unconfigured nvi. I realize now though that you're editing /etc/fstab inside the chroot anyway so that doesn't really save any hassle. I added a note though that nvi is avaible but you may wish to install a different text editor.

Reply:
Yeah, this is mostly just common practice on *other* manual installation guides i've used in the past. (like arch) not a big deal.